CONTRIBUTING.md

```diff
@@ -13,7 +13,7 @@ This section describes the main steps required before you can add your code and
 ### Prerequisites
 * Linux machine with x86-64 architecture with bash installed.
 * Basic tools: curl, tar, make.
-* Python3 and pipenv.
+* Python3 and pipenv. (eg: `python3 -m pip install pipenv --user`)
```

Comment:
I'd avoid providing it as an example here, as we already know that alternative approaches for installing pipenv might not work. I suggest instead mentioning what needs to be run. Something like:
```suggestion
* Python3 and pipenv. (run: `python3 -m pip install pipenv --user`)
```

Reply:
Makes sense. I updated it now.